CONTRIBUTING.md

```diff
@@ -34,20 +34,20 @@ If you're new to the project and want to help, but don't know where to start, we
     1. See the [Development Guide](https://capz.sigs.k8s.io/developers/development.html) for more instructions on setting up your environment and testing changes locally.
 3. Submit a pull request.
     1. All PRs should be labeled with one of the following kinds
-         - `/kind feature` for PRs releated to adding new features/tests
-         - `/kind bug` for PRs releated to bug fixes and patches
-         - `/kind api-change` for PRs releated to adding, removing, or otherwise changing an API
-         - `/kind cleanup` for PRs releated to code refactoring and cleanup
+         - `/kind feature` for PRs related to adding new features/tests
+         - `/kind bug` for PRs related to bug fixes and patches
+         - `/kind api-change` for PRs related to adding, removing, or otherwise changing an API
+         - `/kind cleanup` for PRs related to code refactoring and cleanup
          - `/kind deprecation` for PRs related to a feature/enhancement marked for deprecation.
-         - `/kind design` for PRs releated to design proposals
-         - `/kind documentation` for PRs releated to documentation
-         - `/kind failing-test` for PRs releated to to a consistently or frequently failing test.
+         - `/kind design` for PRs related to design proposals
+         - `/kind documentation` for PRs related to documentation
+         - `/kind failing-test` for PRs related to to a consistently or frequently failing test.
```

Comment:
Hm, seems this could be fixed as well?

```suggestion
         - `/kind failing-test` for PRs related to a consistently or frequently failing test.
```